deployment-templates/compute-engine/server/forseti-instance-server.py

```diff
@@ -193,6 +193,11 @@ def GenerateConfig(context):
 echo "Starting services."
 systemctl start cloudsqlproxy
 sleep 5
+
+echo "Attempting to upgrade database."
```

Comment:
I would be clear that this updating the `schema`, and add `if necessary` at the end.

```
echo "Attempting to update database schema. if necessary."
```

Reply:
Updated.